plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/ModelSourceWrapper.kt

```diff
@@ -42,9 +42,13 @@ internal class ModelSourceWrapper(
     }
   }
 
-  fun setPosition(value: List<Double>) {
-    val property = hashMapOf(Pair(POSITION, Value(value.map { Value(it) })), Pair(URL, Value(url)))
-    val updateModel = hashMapOf(Pair(DEFAULT_MODEL_NAME, Value(property)))
+  fun setPositionAndOrientation(lngLat: List<Double>, orientation: List<Double>) {
+    val property = hashMapOf(
+      POSITION to Value(lngLat.map(::Value)),
+      ORIENTATION to Value(orientation.map(::Value)),
+      URL to Value(url)
```

Comment:
It's a litter wired to update URL in this method, is it needed here?

Reply:
We need to update the source property in a bundle, and a url is required here.